src/content/ui-components/install/astro.mdx

```diff
@@ -0,0 +1,77 @@
+---
+title: Astro
+meta:
+  title: Astro | Tiptap UI Components
+  description: Install and configure Tiptap UI Components in your Astro project.
+  category: UI Components
+---
+
+Use the command below to scaffold a new React project with TypeScript and then add Astro.
```

Comment:
I don't think this command creates a React project with Typescript (in the way that Vite or CRA do), instead, I think it creates a new Astro project.